### PR TITLE
edged implement GetExec funtion

### PR DESCRIPTION
**Pull request by kadisi**
Signed-off-by: kadisi <iamkadisi@163.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind test
> /kind failing-test
> /kind feature

/kind feature
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Refs #1477 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: edge/pkg/edged/edged_pods.go [hunk 1]
```diff
@@ -1244,15 +1244,22 @@ func (e *edged) ServeLogs(w http.ResponseWriter, req *http.Request) {
...
+	}
+	if container == nil {
+		return nil, fmt.Errorf("container not found (%q)", containerName)
+	}
+	return e.streamingRuntime.GetExec(container.ID, cmd, streamOpts.Stdin, streamOpts.Stdout, streamOpts.Stderr, streamOpts.TTY)
 }
 
 func (e *edged) GetAttach(podFullName string, podUID types.UID, containerName string, streamOpts remotecommand.Options) (*url.URL, error) {
-	return nil, nil
+	panic("GetAttach need to be continued")
```
Comment (kevin-wangzefeng) [inline edge/pkg/edged/edged_pods.go:1258]: nit: maybe simply "GetAttach need to be implemented" ?

Comment (kadisi) [reply edge/pkg/edged/edged_pods.go:1258]: ok

**Comment by daixiang0** [comment]
Better title: "Edged: implemt GetExec function"

**Comment by kadisi** [comment]
cc @kevin-wangzefeng @fisherxu

**Pull request merged**